test/end-to-end/depots/depots.page.js

```diff
@@ -45,10 +35,10 @@ class DepotPage {
     FU.buttons.create();
     FU.input('DepotModalCtrl.depot.text', name);
     if (hasWarehouse) {
-      element(by.css('[name="is_warehouse"]')).click();
+      $('[name="is_warehouse"]').click();
```

Comment:
I was thinking as you said to me that `element(...)` has better performances than `$(...)`.

What is the reason of using `$(...)` as replacement of `element(...)` ?

Reply:
Ah, no I was referring to `element(by.id())` which will resolve to `document.getElementById()`.

[`$(x)` is an alias for `element(by.css(x))`](http://www.protractortest.org/#/locators#locators)